visual_testing_wdio/test/specs/advancedMethodTest.spec.js

```diff
@@ -0,0 +1,52 @@
+const
+    { captureAndValidateScreen } = require('../../_utils/imageComparisonUtil'),
+    BlogPage = require('../pageObjects/blogPage'),
+    blogPage = new BlogPage()
+
+
+const screenshotCaptureProperties = {
+    landingPage: [
+        {
+            tolerance: 0.0,
+            screenshotName: 'landingPage'
+        }
+    ],
+    servicesPage: [
+        {
+            tolerance: 1.0,
+            screenshotName: 'servicesPage'
+        }
+    ],
+}
+
+describe('Image comparison with advanced method usage', () => {
+
+    it('Verify landing page is displayed as expected', async () => {
+        await browser.url('https://www.atlantbh.com')
```

Comment:
This should be done in setup phase. You have access to browser object in wdio.conf.js and you should open url from there